src/components/app/App.js

```diff
@@ -88,8 +88,18 @@ class App extends Component {
   // todo 1: добавить здесь функцию onUserAdd
   //  она должна добавлять пользователя в список users в стейте
   //  при добавлении пользователя ему нужно добавить пропертю id, можно по аналогии со строкой 82
-
-
+  onUserAdd=(newUser)=>{
+    this.setState((prevState)=>{
+      return{
+        users:[{
+          ...newUser,
+          id:uniqueId(),
+        },
+            ...prevState.user
```

Comment:
I guess it's not working
Because prevState doesn't contain such property as `user`
And I guess that user's list got only 1 element each time you are adding user
Could you please check it one more time?